examples/vision/yolov8.py

```diff
@@ -0,0 +1,571 @@
+"""
+Title: Efficient Object Detection with YOLOV8 and KerasCV
+Author: [Gitesh Chawda](https://www.linkedin.com/in/gitesh-ch/)
+Date created: 2023/05/15
+Last modified: 2023/05/15
+Description: Train custom YOLOV8 object detection model with KerasCV.
+"""
+
+"""
+KerasCV is a deep learning library built on top of Keras, a popular deep learning
```

Comment:
I think some of this information is implied (e.g. we can probably assume folks know what Keras is)

Perhaps something like "KerasCV is an extension of Keras for computer vision tasks. In this example, we'll see how to train a YOLOV8 object detection model using KerasCV."